tests/library/browsercontext-cookies.spec.ts

```diff
@@ -178,7 +178,7 @@ it('should get cookies from multiple urls', async ({ context, browserName, isWin
     expires: -1,
     httpOnly: false,
     secure: true,
-    sameSite: 'None',
+    sameSite: (browserName === 'webkit' && platform === 'darwin' && parseInt(os.release(), 10) >= 24) ? 'Lax' : 'None',
```

Comment:
let's extract it in a fixture and use it in all affected tests? something like `sameSiteStoredValueForNone`